_layouts/layout.htm

```diff
@@ -7,11 +7,9 @@
 	{% include _footer.htm %}
 
 	<!--   Core JS Files   -->
-	<script src="/assets/js/core/jquery.3.2.1.min.js" type="text/javascript"></script>
+	<script src="/assets/js/core/jquery-3.4.1.min.js" type="text/javascript"></script>
```

Comment:
Note: JQuery is also being updated.  Does this affect/break anything?

Reply:
As far as my local tests goes, Everything seems to function as before. Bootstrap isn't reporting any problems nor is NowUI. Everything looks correct.